src/vs/editor/contrib/suggest/suggestWidget.ts

```diff
@@ -9,6 +9,7 @@ import { createMatches } from 'vs/base/common/filters';
 import * as strings from 'vs/base/common/strings';
 import { Event, Emitter } from 'vs/base/common/event';
 import { onUnexpectedError } from 'vs/base/common/errors';
+import { ClassName } from 'vs/editor/common/model/intervalTree';
```

Comment:
This dependency shouldn't exist. Define a suggest-widget-private style and simply refer to its classname further down.

Reply:
:top: I guess that answers https://github.com/microsoft/vscode/pull/78092#issuecomment-515993758. No need to externalise/re-use